### PR TITLE
Homepage

### PR DESCRIPTION
**Pull request by Lauraa1003**
Test URLs:
- Before: https://main--www--headwire-edge-delivery.hlx.page/
- After: https://homepage--www--headwire-edge-delivery.hlx.page/

### CODE REVIEW

File: styles/styles.css [hunk 1]
```diff
@@ -11,241 +11,30 @@
...
-  --heading-font-size-xxl: 48px;
-  --heading-font-size-xl: 40px;
-  --heading-font-size-l: 32px;
-  --heading-font-size-m: 24px;
-  --heading-font-size-s: 20px;
-  --heading-font-size-xs: 18px;
+  /* font-family doesn't work without capital */
+  
+   /* stylelint-disable-next-line value-keyword-case */
+  --body-font-family: "Raleway", -apple-system, BlinkMacSystemFont, "Segoe UI", roboto, "Helvetica Neue", arial, "Noto Sans", sans-serif, "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol", "Noto Color Emoji";
```
Comment (icaraps) [inline styles/styles.css]: You'll also have to add the raleway font https://fonts.googleapis.com/css2?family=Raleway%3Aital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap

File: blocks/cards/cards.js [hunk 2]
```diff
@@ -15,4 +15,19 @@ export default function decorate(block) {
   ul.querySelectorAll('img').forEach((img) => img.closest('picture').replaceWith(createOptimizedPicture(img.src, img.alt, false, [{ width: '750' }])));
   block.textContent = '';
   block.append(ul);
+
+  // add svg icons
+  const cardListItems = document.querySelectorAll('.cards.light.block li');
```
Comment (icaraps) [inline blocks/cards/cards.js]: We can remove that

**Comment by icaraps** [comment]
Screenshot 2023-09-15 at 17 10 07

There seems to be an error in cards

File: blocks/cards/cards.js (hunk 2 above)
Comment (Lauraa1003) [reply blocks/cards/cards.js]: Removed

File: styles/styles.css (hunk 1 above)
Comment (Lauraa1003) [reply styles/styles.css]: added font

**Pull request merged**